src/status_im/ui/screens/home/styles.cljs

```diff
@@ -185,7 +185,10 @@
 (def action-button-container
   {:position    :absolute
    :align-items :center
-   :bottom      (+ tabs.styles/tabs-diff 6)
+   :bottom      (+ tabs.styles/tabs-diff (cond
+                                           platform/ios? 16
```

Comment:
@errorists Does that sound right to you? Different padding based on the platform?

Reply:
No, not at all unless it's required to make it appear in the right place. I tested it and on iOS it's correct.

Reply:
@jeluard even though `:bottom` value is different for each platform, margin is 16.  I tested it on iOS and Android, but not on desktop (couldn't run dev build).

Reply:
I made it 16 margin for iOS and Android and left desktop value unchanged.